cloud/aws/apigateway/modules.tf

```diff
@@ -0,0 +1,7 @@
+module "filter-tags" {
+  source = "github.com/claranet/terraform-signalfx-detectors.git//common/filter-tags"
+
+  filter_defaults        = "filter('env', '${var.environment}') and filter('sfx_monitored', 'true')"
```

Comment:
Datadog does not retrieve tags from stages for these metrics so may be it is also the case for signalfx and in this case we cannot use convention tagging

Reply:
we added tag synching for API Gateway last year